src/pal/inc/pal.h

```diff
@@ -3340,6 +3340,14 @@ PALAPI
 LPCVOID
 PAL_GetCoreClrModuleBase();
```

Comment:
We should delete this method. The one remaining use can be replaced by PAL_GetPalModuleBase.